tests/infer/test_gradient.py

```diff
@@ -1,5 +1,7 @@
 from __future__ import absolute_import, division, print_function
 
+import logging
```

Comment:
I'm used to run `pytest -s` to see this test output. How can I invoke `pytest` after this PR to see the logging output? Is it something like 'PYRO_LOG_LEVEL=1 pytest`?

Reply:
The test root logger has the default level set to `DEBUG` (we can change this later depending on some environment flag) so you shouldn't need to do anything else. `pytest -s` should work fine.